packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -36,7 +36,7 @@ class BuildInfo {
     List<String>? dartExperiments,
     required this.treeShakeIcons,
     this.performanceMeasurementFile,
-    this.packagesPath = '.packages', // TODO(zanderso): make this required and remove the default.
+    this.packagesPath = '.dart_tool/package_config.json', // TODO(zanderso): make this required and remove the default.
```

Comment:
Maybe this hard-coded path separator is leading to coverage collection failing on Windows?

Reply:
odd that it is failing only for windows. Really, you should be able to use `/` conditionally ...

Perhaps the coverage code is doing some string matching instead of using package:path ?

Reply:
Using the platform separator will make the constructor non-const (which might not be a problem).

Cursory inspection doesn't show anything in package:coverage that should depend on backslash on windows...

Reply:
I'll debug this one today, I use a windows machine

Reply:
Gentle ping

This is blocking https://dart-review.googlesource.com/c/sdk/+/231101 .